frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-body.tsx

```diff
@@ -28,12 +29,29 @@ export const BreakdownCardBody: React.FC<BreakdownBodyProps> = ({
       </EmptyState>
     );
   }
+  if (capacityUsed === '0') {
+    return (
+      <EmptyState variant={EmptyStateVariant.full}>
+        <Title className="graph-empty-state__title text-secondary" size="sm">
+          Not enough usage data
+        </Title>
+      </EmptyState>
+    );
+  }
 
-  const available = getCapacityValue(cephUsed, cephTotal, humanize);
-  const usedCapacity = `${humanize(cephUsed).string} used of ${humanize(cephTotal).string}`;
+  const available = getCapacityValue(capacityUsed, capacityTotal, humanize);
```

Comment:
This can be inside if block below because not being calculated for noobaa breakdown.